test/e2e/framework.go

```diff
@@ -163,6 +164,7 @@ const (
 var (
 	errConnectionLost = fmt.Errorf("http2: client connection lost")
 	clickHousePodName = fmt.Sprintf("%s-0-0-0", clickHousePodNamePrefix)
+	mutex             sync.Mutex
```

Comment:
I feel `mutex` is a vague name and this Mutex is not used in other e2e tests. I will recommend putting it under throughputanomalydetection_test.go